app/src/main/java/io/homeassistant/companion/android/notifications/MessagingManager.kt

```diff
@@ -432,6 +435,19 @@ class MessagingManager @Inject constructor(
                             }
                         }
                     }
+                    COMMAND_PERSISTENT_CONNECTION -> {
+                        if (!jsonData[PERSISTENT].isNullOrEmpty()) {
```

Comment:
Lets add 1 more validation step to ensure the user is only entering accepted values, otherwise we should also post the notification to the device as the data is still not valid